src/main/kotlin/com/terraformation/backend/search/field/UriField.kt

```diff
@@ -0,0 +1,48 @@
+package com.terraformation.backend.search.field
+
+import com.terraformation.backend.db.collation
+import com.terraformation.backend.i18n.currentLocale
+import com.terraformation.backend.search.FieldNode
+import com.terraformation.backend.search.SearchFilterType
+import com.terraformation.backend.search.SearchTable
+import java.net.URI
+import java.util.EnumSet
+import org.jooq.Condition
+import org.jooq.Field
+import org.jooq.impl.DSL
+
+/** Search field for urls. */
+class UriField(
+    override val fieldName: String,
+    override val databaseField: Field<URI?>,
+    override val table: SearchTable,
+    override val nullable: Boolean = true,
+) : SingleColumnSearchField<URI>() {
+  override val localize: Boolean
+    get() = false
+
+  override val supportedFilterTypes: Set<SearchFilterType>
+    get() = EnumSet.of(SearchFilterType.Exact, SearchFilterType.PhraseMatch)
+
+  override fun getCondition(fieldNode: FieldNode): Condition {
+    val valuesRegex = fieldNode.values.map { "$it" }
```

Comment:
What is this doing? The list of values is already `List<String?>` so this seems like it would be a no-op for actual values, but would turn null values into `"null"` which doesn't seem like what we'd want.

The fact that the variable name says this is a regex, but the value is a list of non-regex strings, makes me think the intention was something else here.

Reply:
Ahhh I left this with some escaped operations earlier. I needed to modify this to be filter non nulls basically.